app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elf: list[Elf]) -> None:
+    for elf_one in elf:
+        elf_one.play_elf_song()
```

Comment:
why don'y you use `elves` as a list and simple `elf` in loop

Reply:
got it